st2auth/st2auth/config.py

```diff
@@ -59,6 +59,8 @@ def _register_app_opts():
     api_opts = [
         cfg.ListOpt('allow_origin', default=['http://localhost:3000'],
             help='List of origins allowed'),
+        cfg.BoolOpt('serve_webui_files', default=True,
```

Comment:
Would if we should actually move it to st2common.config since we already have some common API options there...